documentation/BrowserChannel.md

```diff
@@ -24,39 +24,45 @@ Also, this component create notifications for every connected client at any time
 This small and documented client illustrates the workflow.
 
 ```javascript
-var io = require("socket.io-client");
+var io = require("ws");
 
 // The client starts the communication
 //
-var socket = io.connect("http://localhost:8081/browserChannel");
+var socket = new ws("ws://localhost:8081/browserChannel");
 
 // When the connection is done, the client tells to the flow manager its id
-//
-socket.on("connect", function () {
-    console.log("## Socket connected");
-    socket.send("org.chrome.cloud4chrome");
-});
 
-// Right after sending the id to the flow manager, the server will return back
-// the current settings in the system (if any)
-//
-socket.on("connectionSucceeded", function (settings) {
-    console.log("## Received the following settings: " + JSON.stringify(settings));
+socket.on("open", function () {
+    console.log("## Socket connected");
+    socket.send(JSON.stringify({
+        type: "connect",
```

Comment:
Ah, I'm sorry - the call site is here:
https://github.com/amb26/universal/blob/GPII-1318/gpii/node_modules/settingsHandlers/src/WebSocketsSettingsHandler.js#L143
I've added docs at https://github.com/amb26/kettle/blob/KETTLE-32/docs/RequestHandlersAndApps.md#websockets-request-components-of-type-kettlerequestws for the "sendTypedMessage" method which I had forgotten to document, along with "sendMessage". You can see that we demultiplex them in the test cases like this which is pretty easy to do:

https://github.com/amb26/universal/blob/GPII-1318/gpii/node_modules/flowManager/test/shared/BrowserChannelTestDefs.js#L53

But.... you presumably know all this since you have bound to my branch for your browser plugin work?